scripts/gef.sh

```diff
@@ -3,17 +3,22 @@
 set -e
 
 branch="master"
-test "$1" == "dev" && branch="dev"
+
+if [ "$1" = "dev" ]; then
+    branch="dev"
+    echo "set branch to dev"
+fi
 
 # Backup gdbinit if any
-test -f "${HOME}/.gdbinit" && mv "${HOME}/.gdbinit" "${HOME}/.gdbinit.old"
+if [ -f "${HOME}/.gdbinit" ]; then
+    mv "${HOME}/.gdbinit" "${HOME}/.gdbinit.old"
+fi
 
 # Get the hash of the commit
-ref=$(curl --silent https://api.github.com/repos/hugsy/gef/git/ref/heads/${branch} | grep '"sha"' | tr -s ' ' | cut -d ' ' -f 3 | tr -d "," | tr -d '"')
+ref=$(wget -q -O- https://api.github.com/repos/hugsy/gef/git/ref/heads/${branch} | grep '"sha"' | tr -s ' ' | cut -d ' ' -f 3 | tr -d "," | tr -d '"')
```

Comment:
Some distros do not have `wget` by default, `curl` is usually always found and I'm surprised Ubuntu changed that.
Can you change your PR to determine whether `curl` or `wget` exist on the system, and then use the found command to download gef ?

Reply:
We could make it so the script takes an arg e.g. `--curl` if you want to use curl or vice versa.

Reply:
True, but we still need a default case, and we might as well try to determine it dynamically.